docs/build-on-linea/tooling/index.md

```diff
@@ -3,8 +3,17 @@ title: Tooling and infrastructure
 sidebar_position: 3
 ---
 
+# Disclaimer
```

Comment:
Rather than a whole section, consider using the warning styling instead under the `Explore developer tooling and infrastructure` heading?

And the same with the other file?

```
:::warning

text

:::
```

Reply:
done